src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```
@@ -254,6 +268,32 @@ protected function createConfigurationDefinition()
                 ->getOption(),
             (new FixerOptionBuilder('operators', 'Dictionary of `binary operator` => `fix strategy` values that differ from the default strategy.'))
                 ->setAllowedTypes(['array'])
+                ->setAllowedValues([function ($option) {
+                    foreach ($option as $operator => $value) {
+                        if (!in_array($operator, self::$supportedOperators, true)) {
+                            throw new InvalidOptionsException(
```

we shall throw this exception (we do it in other places), and then, in `AbstractFixer::configure`, it's converted to `InvalidFixerConfigurationException`